### ISSUE TITLE
关于ptuning部分的input和label构建的问题

### ISSUE THREAD

**Issue by TheHonestBob** [THUDM/ChatGLM-6B#356]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

无

### Expected Behavior

无

### Steps To Reproduce

无

### Environment

```markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :
无
```

### Anything else?

我看ptuning的示例代码中，构建模型输入的input是数据中的content，而模型label是数据中的summary，但是对于chatglm模型，通过modeling_chatglm.py中的 def chat()函数中的代码可知，原始的chatglm训练时的label是input+label，请问ptuning中并未采用此方式，是否会有一定影响，或者说是我误解了什么，感谢回复

**Comment by smartparrot** [comment]
同问，训练微调模型和训练预训练模型，输入难道不同？

**Comment by duzx16** [comment]
在`preprocess_function_train`中输入也是input+label。`preprocess_function_eval`里，evaluate的时候当然不能给模型提供label
